File: result/translate/chatgpt/java2python/python/TEMPLE_OFFERINGS.py
def f_gold(n, templeHeight):
    sum = 0
    for i in range(n):
        left = 0
        right = 0
        j = i - 1
        while j >= 0:
            if templeHeight[j] < templeHeight[j + 1]:
                left += 1
            else:
                break
            j -= 1
        
        j = i + 1
        while j < n:
            if templeHeight[j] < templeHeight[j - 1]:
                right += 1
            else:
                break
            j += 1
        
        sum += max(right, left) + 1
        
    return sum

File: result/translate/starchat/python2java/java/TEMPLE_OFFERINGS.java
public static int f_gold ( int n, int[] templeHeight ) {
        int sum = 0;
        for ( int i = 0; i < n; i++ ) {
            int left = 0, right = 0;
            for ( int j = i - 1; j >= 0 && templeHeight[j] < templeHeight[j + 1]; j-- ) {
                left++;
            }
            for ( int j = i + 1; j < n && templeHeight[j] < templeHeight[j - 1]; j++ ) {
                right++;
            }
            sum += Math.max(right, left) + 1;
        }
        return sum;
    }